domain/AdminProperties.go

```diff
@@ -315,6 +316,7 @@ func (d *Domain) AdminProperties(in *AdminPropertiesIn) (out AdminPropertiesOut)
 		fallthrough
 	case zCrud.CmdList:
 		r := rqProperty.NewProperty(d.PropOltp)
+		out.PropertiesWithNote = r.FindByPaginationWithNote(&AdminPropertiesMeta, &in.Pager, &out.Pager)
 		out.Properties = r.FindByPagination(&AdminPropertiesMeta, &in.Pager, &out.Pager)
```

Comment:
ini saja yg langsung diisi dari `r.FindByPaginationWithNote(&AdminPropertiesMeta, &in.Pager, &out.Pager)`